sklearn/model_selection/_validation.py

```diff
@@ -965,9 +965,19 @@ def _check_is_permutation(indices, n_samples):
 def permutation_test_score(estimator, X, y, *, groups=None, cv=None,
                            n_permutations=100, n_jobs=None, random_state=0,
                            verbose=0, scoring=None):
-    """Evaluate the significance of a cross-validated score with permutations
+    """Evaluates the significance of a cross-validated score using permutations
 
-    Read more in the :ref:`User Guide <cross_validation>`.
```

Comment:
We might still want to keep the link to the user manual and the end of the docstring maybe?

Reply:
[Cross_validation ](https://scikit-learn.org/stable/modules/cross_validation.html#cross-validation)never talks about `permutation_test_score`. In fact it is never mentioned in the user guide at all. If we wanted to add a section in the user guide it would probably fit somewhere in ['Model selection and evaluation'](https://scikit-learn.org/stable/model_selection.html). I'm happy to add a section but I won't have much to say and it would be the same as what I've added in the example: https://github.com/scikit-learn/scikit-learn/pull/17385